scripts/mergeswag.sh

```diff
@@ -3,13 +3,17 @@
 # Mergeswag.sh merges JSON-encoded Swagger specification files into one mongo file
 # called swagger.json.
 
-[ -z "$1" ] && echo >&2 "Please specify a folder to search for .swagger.json files" && exit 1
+[ -z "$1" ] && [[ "$1" =~ ^(1|2)$ ]] && echo >&2 "Please specify a valid API version number for merged swagger.json file" && exit 1
 
-[ -z "$2" ] && [[ "$2" =~ ^(1|2)$ ]] && echo >&2 "Please specify a valid API version number for merged swagger.json file" && exit 1
+[ -z "$2" ] && echo >&2 "Please specify at least one folder to search for .swagger.json files" && exit 1
 
 set -euo pipefail
 
-folder="$1"
+src=$(mktemp -d)
+
+for folder in "${@:2}"; do
+    find "$folder/" -name '*.swagger.json' -exec cp {} $src \;
+done
 
 export TITLE="API Reference"
 export VERSION="$2"
```

Comment:
Because we have swapped order of arguments this should be updated accordingly:
```suggestion
export VERSION="$1"
```
Please check if there are other places, I didn't check everything.